tsconfig.json

```diff
@@ -24,8 +24,15 @@
     },
     "typeRoots": ["./src/types", "./node_modules/@types"]
   },
-  "include": ["src"],
-  "exclude": ["dist", "node_modules"],
+  "include": [
+    "src", 
+    "immutable", 
```

Comment:
Will this also change the bundled files to include everything?

Reply:
No, bundler collect files from entry. but change `tsc` behavior like if you run it directly it will compile all these folders. That's why I use them for typing check as well